### PR TITLE
Add PiJuice support

### PR DESCRIPTION
**Pull request by smallhacks**
Get PiJuice status (status_error, is_fault, battery_status, charge_level_error, charge_level) and show them in the plugin table.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -163,6 +172,15 @@
 
     $table->add_data(array(get_string('sdcardavailablespace', 'tool_moodlebox'), display_size($sdcardfreespace) .
             ' (' . 100 * round($sdcardfreespace / $sdcardtotalspace, 3) . '%)'));
+    if ($pijuicepackage) {
+        if ($pijuicestatus['status_error'] == "NO_ERROR") {
+            $table->add_data(array(get_string('pijuicebatterychargelevel', 'tool_beekeebox'), $pijuicestatus['charge_level'] . '%'));
```
Comment (martignoni) [inline index.php]: String is called from invalid package. Should be 'tool_moodlebox'.

**Comment by smallhacks** [comment]
@martignoni 
I think there was an error in the Travis CI confiiguration that you resolved with #5abe11f, can you restart the build to see if the test pass, please?

**Comment by martignoni** [comment]
It still fails, since during the CI the pull request is done after the .travis.yml config file is read. Can you please rebase your PR?

**Comment by smallhacks** [comment]
Yes, you're right. It's done now.

**Comment by martignoni** [comment]
### Test results

#### No battery temp indicated 

See screenshot. This is strange, as `pijuice_cli` utility shows it correctly.
not-temp

Any comment?

#### Communication with PiJuice problem

After PiJuice normal installation, I had to manually add line `dtoverlay=i2c-bcm2708` to `/boot/config.txt` and reboot to make it work (as indicated [in this comment](https://github.com/PiSupply/PiJuice/issues/105#issuecomment-386895587)). Otherwise, I got communication errors. This is not directly related to this PR, but makes me uncomfortable with the usability. 

Maybe we should add some documentation to help the user. What do you think ?

**Comment by smallhacks** [comment]
> No battery temp indicated
Sorry, it was a mistake (I don't currently have any PiJuice to test it properly). It should be fixed now.

> Communication with PiJuice problem
It is weird since I never had to do that. The comment you talk about seems to be related to the 1.2 firmware version. I now there is at least a 1.3 version already, did you try it ?

**Comment by martignoni** [comment]
Thanks for the fix.

Re communication problem: I used indeed a 1.3 firmware version. Maybe my PiJuice was not properly configured from the beginning. I'll test again.

**Pull request merged**